src/data/pnat/basic.lean

```diff
@@ -279,16 +279,19 @@ def mod (m k : ℕ+) : ℕ+ := (mod_div m k).1
 -/
 def div (m k : ℕ+) : ℕ  := (mod_div m k).2
 
-theorem mod_add_div (m k : ℕ+) : (m : ℕ) = (mod m k) + k * (div m k) :=
+theorem mod_add_div (m k : ℕ+) : ((mod m k) : ℕ) + k * (div m k) = (m : ℕ) :=
```

Comment:
Does this work? I suspect most of the casts are implied, but I'm not certain which
```suggestion
theorem mod_add_div (m k : ℕ+) : (mod m k + k * div m k : ℕ) = m :=
```

Reply:
This also seems to work (at least locally).